CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.25.1 (Mar 28, 2022)
+
+BUG FIXES:
+
+* Fix retrieval_cache_period_seconds to be set to 0 for artifactory_*_repository resources. [GH-373]
```

Comment:
The change affects only remote repos, right? Let's be more specific in the changelog.

Reply:
i see "retrieval_cache_period_seconds" ... is available for both remote and virtual repos.

Reply:
The change affects `RemoteRepositoryBaseParams` only.

Reply:
will update the change log to be specific.